bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/configuration/ServerConfiguration.java

```diff
@@ -48,6 +48,8 @@ public void setWorkspaceRoot(final String workspaceRoot) {
          .ifPresent(uri -> this.workspaceRootURI = uri);
    }
 
+   public String getWorkspaceUISchemaPath() { return workspaceRootURI.toFileString().concat(".ui-schemas/"); }
```

Comment:
This is fine for now but I believe we should discuss more flexibility in the future if needed, e.g., at least having a setter for that so that we can read it from the command line or change it some other way. My main line of thinking is that we probably do not always want to have that on the workspace level.

Reply:
You're right, I used the same logic now to set the ui schema folder path as for the workspace root and detached it from the workspace root. It is now possible to set any valid file path as ui schema folder via the command line argument `-u`.